addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_document.py

```diff
@@ -411,10 +418,26 @@ def _get_recipient_values(self, partner, is_simplified=False):
 
         return {'recipient': recipient_values}
 
+    def _get_refunded_values(self, values):
+        if not values.get('is_refund'):
+            return {}
+        refunded_doc = values['refunded_doc']
+        refunded_name = values['refunded_name']
+        if refunded_doc:
+            sequence, number = refunded_doc._get_tbai_sequence_and_number()
+        else:
+            sequence, number = self._get_tbai_seq_from_name(refunded_name)
```

Comment:
Not sure but maybe we can just deprecate `_get_tbai_sequence_and_number` (we still need it for the XML but it does not really do more than the new function)
```suggestion
        refunded_name = refunded_doc.name if refunded_doc else values['refunded_name']
        sequence, number = self._get_tbai_seq_from_name(refunded_name)
```